### ISSUE TITLE
Scraper for VD fails when using bash

### ISSUE THREAD

**Issue by baryluk** [openZH/covid_19#205]
It works with dash (default `/bin/sh` on Debian/Ubuntu):

```
VD
Scraped at: 2020-03-26T10:05:28+01:00
Date and time: 24.03.2020
Confirmed cases: 2234
Deaths: 21
```

but not with bash

```
VD
Scraped at: 2020-03-26T10:05:32+01:00
Date and time: Date\tHospitalisations
Confirmed cases: l'h\u00f4pital\tD\u00e9c\u00e8s\tTotal
Deaths: de
```

To be investigated.

**Comment by baryluk** [comment]
@zdavatz Out of curiosity which OS, Distro are you using that defaults `/bin/sh` to bash?

**Comment by baryluk** [comment]
BTW. It is probably easier to just port it to Python, like all the other scrapers, and not deal with this dash / bash difference. (it could be a bug in one of the shells tho. who knows).

**Comment by zdavatz** [comment]
> @zdavatz Out of curiosity which OS, Distro are you using that defaults `/bin/sh` to bash?

Gentoo Linux ;)

**Comment by baryluk** [comment]
OK. I was expecting that.

Looks like using `echo -e` will help.

**Comment by metaodi** [comment]
Closing since I merged #236
